### PR TITLE
Support for other views filter in sectioned view

### PR DESCRIPTION
**Pull request by pfranc**

### CODE REVIEW

File: builder/converter/templates/sectioned_view_other_views_filter_template.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<hudson.views.OtherViewsFilter  plugin="view-job-filters@1.26">
+  <includeExcludeTypeString>includeMatched</includeExcludeTypeString>
+  <otherViewName>NAME</otherViewName>
```
Comment (michalvanco) [inline builder/converter/templates/sectioned_view_other_views_filter_template.xml]: is the `NAME` a default value when you add such an empty filter?

**Comment by michalvanco** [comment]
Include example at tests/fixtures/sectioned_view.yaml + related test

**Comment by pfranc** [comment]
Other views filter default value changed. Test added

**Comment by michalvanco** [comment]
Looks OK, thanks!

**Pull request merged**